dataland-frontend/src/components/forms/parts/YesNoComponent.vue

```diff
@@ -31,12 +32,23 @@ export default defineComponent({
   props: {
     name: {
       type: String,
+      default: "",
     },
     info: {
       type: String,
+      default: "",
     },
     displayName: {
       type: String,
+      default: "",
+    },
+    radioButtonsOptions: {
+      type: Array as () => Array<string>,
```

Comment:
We have type data called "YesNoNa" I thought that if we want to use such a type, we can by add another element "NA" to props radioButtonsOptions or if in some place we will use the component to display other radiobuttons, e.g. true false

Reply:
> Component should be named differently probably, e.g RadiobuttonsList

Actually I really like this idea!  How about `RadioButton.vue` ?

Because e.g. in `BasicInformationFields.vue` you use it with:
`:radioButtonsOptions="['Deviation', 'NoDeviation']"`

So there it is no actual YesNo-component anymore.

Reply:
RadioButtonsGroup.vue ?

Reply:
Ok 👍